Orange/widgets/data/owtransform.py

```diff
@@ -18,29 +17,29 @@ class OWTransform(OWWidget):
 
     class Inputs:
         data = Input("Data", Table, default=True)
-        preprocessor = Input("Preprocessor", Preprocess)
+        template_data = Input("Template Data", Table)
 
     class Outputs:
         transformed_data = Output("Transformed Data", Table)
 
     class Error(OWWidget.Error):
-        pp_error = Msg("An error occurred while transforming data.\n{}")
+        error = Msg("An error occurred while transforming data.\n{}")
 
     resizing_enabled = False
     want_main_area = False
 
     def __init__(self):
         super().__init__()
         self.data = None
-        self.preprocessor = None
+        self.template_data = None
```

Comment:
Could you perhaps store just the domain (in `template_domain` instead of `template_data`)? It would save some space and contribute to clarity -- it would be more obvious (from the code if not from the input signal type) that the widget needs just the domain.